install.yml

```diff
@@ -95,7 +95,6 @@
     - { role: arteria-sequencing-report-ws, tags: arteria-sequencing-report }
     - { role: snpseq-metadata-service, tags: snpseq-metadata-service }
     - { role: standalone_scripts, tags: standalone_scripts }
-    - { role: delivery, tags: delivery }
```

Comment:
what was this tag about?

Reply:
I believe it's Uppsala's old delivery role?

Reply:
Yeah, it was for deliveries using mover, which we no longer do.